README.md

```diff
@@ -22,31 +20,31 @@ Thanks!
 If you're reading this at https://github.com/rails-api/active_model_serializers you are
 reading documentation for our `master`, which is not yet released.
 
-Please see below for the documentation relevant to you.
-
 - [0.10 (0-10-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-10-stable)
 - [0.10.6 (latest release) Documentation](https://github.com/rails-api/active_model_serializers/tree/v0.10.6)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
   - [Guides](https://github.com/rails-api/active_model_serializers/tree/v0.10.6/docs)
 - [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
 - [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+[![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
 
 
 ## Status of AMS
 
-*Status*:
+### *Status*:
 
 - ❗️ All existing PRs against master will need to be closed and re-opened against 0-10-stable, if so desired
 - ❗️ Master, for the moment, won't have any released version of AMS on it.
+- :eyes: See below for [alternatives](#alternatives)
 
-*Changes to 0.10.x maintenance*:
+
+### *Changes to 0.10.x maintenance*:
 
 - The 0.10.x version has become a huge maintenance version.  We had hoped to get it in shape for a 1.0 release, but it is clear that isn't going to happen.  Almost none of the maintainers from 0.8, 0.9, or earlier 0.10 are still working on AMS. We'll continue to maintain 0.10.x on the 0-10-stable branch, but maintainers won't otherwise be actively developing on it
```

Comment:
would you mind fixing the typo whilst here? s/version/burden? and add a period to end of bullet line?